### PR TITLE
Ao html format

### PR DESCRIPTION
**Pull request by AnamarisOrtiz**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,8 +1,35 @@
...
+    <link rel="stylesheet" href="./assets/css/style.css">
+</head>
+<body>
+<header>
+    <h1>What's for dinner?</h1>
+</header>
+    <div id="search-bar">
+
+    </div>
+    <div id="container-recepie-results">
```
Comment (egodschalk) [inline index.html]: recipe spelling

File: index.html [hunk 2]
```diff
@@ -1,8 +1,35 @@
...
+        <section id="recepie-results">
+        </section>
+     </div>
+    <div id="container-viedo-results">
```
Comment (egodschalk) [inline index.html]: video spelling

**Review by egodschalk — CHANGES_REQUESTED** [review]
Spelling of recipe on line 20/21

**Review by AnamarisOrtiz — COMMENTED** [review]
changes made

**Pull request closed without merging**